src/types/sdkDapp.types.ts

```diff
@@ -21,7 +21,9 @@ export type { OnProviderLoginType } from '@multiversx/sdk-dapp/types';
 export type { SignModalPropsType } from '@multiversx/sdk-dapp/types/transactions.types';
 export type { TokenType } from '@multiversx/sdk-dapp/types/tokens.types';
 export type { MultiSignTransactionType } from '@multiversx/sdk-dapp/types/transactions.types';
+export type { CollectionType } from '@multiversx/sdk-dapp/types/tokens.types';
 export { NftEnumType } from '@multiversx/sdk-dapp/types/tokens.types';
+export { EsdtEnumType } from '@multiversx/sdk-dapp/types/tokens.types';
```

Comment:
Declare these two with `export type` as well, for consistency. 👍

Reply:
export type works only on types, not enums. EsdtEnumType is an enum